cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +202,89 @@
+# cost (SXG certificates cost money and we don't need each webpackager instance
+# to have a separate SXG certificate) and prevents the webpkgserver instances
+# from bombarding your Certificate Authority with certificate requests. Note
+# this setup requires you to locate the certificate file on a shared filesystem
+# accessible by all webpkgserver instances.
+#
+# For the full ACME spec, see:
+#   https://tools.ietf.org/html/rfc8555 
+[SXG.ACME]
+  # Enable enables Web Packager to attempt to auto renew certificates using
+  # ACME. Required when ACME is enabled.
+  #Enable = true
+
+  # The ACME discovery URL. It is specified by the Certificate Authority that
+  # doles out your certificates. As of August 2020, only DigiCert supports
+  # automatic renewals of signed exchange certificate:
+  # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
+  DiscoveryURL = '<Your Discovery URL>'
+
+  # The email address registered to the Certificate Authority for your signed
+  # exchange certificates.
+  Email = 'user@example.com'
+
+  # For the remaining configuration items, it's important to understand the
+  # different challenges employed as part of the ACME protocol:
+  # https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#identifier-validation-challenges
+  # https://letsencrypt.org/docs/challenge-types/
+  # https://certbot.eff.org/docs/challenges.html
+  #
+  # Note that you don't need to have all the challenges configured, it's
+  # typically sufficient to have one configured. It isn't required to have
+  # exactly one parameter. They can all be set and it's up to the library to
+  # decide which method to use and to use the other settings as backup. For
+  # wildcard certificates, only DNSProvider is a supported method of validation,
+  # the others cannot be used. See below.
+
+  # The http server root directory where the ACME http challenge token
+  # be deposited. Note that you may need to do some configuration work to
+  # get this setup to work where multiple instances of Web Packager is running.
+  # For example:
+  # https://community.letsencrypt.org/t/how-to-nginx-configuration-to-enable-acme-challenge-support-on-all-http-virtual-hosts/5622/3
+  HTTPWebRootDir = '/path/to/www_root_dir'
+
+  # The port used by the Web Packager to respond to the HTTP challenge
+  # issued as part of ACME protocol. Note that if your setup only opens up
+  # certain ports, you may need to do a configuration change where you forward
+  # requests to this port using proxy_pass, for example:
+  # https://medium.com/@dipeshwagle/add-https-using-lets-encrypt-to-nginx-configured-as-a-reverse-proxy-on-ubuntu-b4455a729176
+  HTTPChallengePort = 5002
+
+  # The port used by Web Packager to respond to the TLS challenge issued as part
+  # of the ACME protocol.
+  TLSChallengePort = 5003
+
+  # The DnsProvider to be used in fulfilling the ACME DNS challenge.
```

Comment:
* s/DnsProvider/DNS provider/
* s/in/for/

Reply:
Done.